src/overlays/gc/gamefix.c

```diff
@@ -1,11 +1,73 @@
-#include "common.h"
+#include "gc/gamefix.h"
 
-#pragma GLOBAL_ASM("asm/nonmatchings/overlays/gc/gamefix/gcgamefix_entrypoint_0.s")
+void gcgamefix_entrypoint_0(void)
+{
+    func_808000D0_gcgamefix();
+    func_8080011C_gcgamefix();
+    func_80800164_gcgamefix();
+    func_800D1864(0x4B, 0, 0);
+    func_800D1864(0x4C, 0, 0);
+}
 
-#pragma GLOBAL_ASM("asm/nonmatchings/overlays/gc/gamefix/gcgamefix_entrypoint_1.s")
+void gcgamefix_entrypoint_1(s32 arg0)
+{
+    u32 temp_s1;
+    u32 temp_s1_2;
+    u32 var_s0;
 
-#pragma GLOBAL_ASM("asm/nonmatchings/overlays/gc/gamefix/func_808000D0_gcgamefix.s")
+    var_s0 = 0;
+    do {
+        //Get the number of Jinjos collected for a certain type
+        temp_s1 = func_800D1338(var_s0);
+        //Compare collected number of jinjos to the total number of jinjos for a type
+        if (func_800D129C(var_s0) == temp_s1)
+        {
+            temp_s1_2 = var_s0 + 0x51;
```

Comment:
This 0x51 should be a jiggy enum, it'd make it clearer that these are the last 9 jiggies of the 90 in the game.